website/blog/2021-09-29-release-apache-apisix-2.10-en.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Release Apache APISIX 2.10.0"
+author: "Zexuan Luo"
+authorURL: "https://github.com/spacewander"
+authorImageURL: "https://avatars.githubusercontent.com/u/4161644?v=4"
+keywords:
+- APISIX
+- Apache APISIX
+- Release Notes
+description: Apache APISIX version 2.10 is released!
+tags: [Release]
+---
+
+> Apache APISIX version 2.10 is released!
+
+<!--truncate-->
+
+Apache APISIX version 2.10 is officially released! 🎉 This is the first LTS release of Apache APISIX with support for 10+ new features and new plugins. Have a quick read to learn about the new features in version 2.10!
+
+## Milestone: The First LTS Telease
+
+This 2.10.0 release is a milestone for Apache APISIX, as Apache APISIX 2.10.0 is our first LTS (Long Time Support) release.
+
+We will be releasing subsequent patch versions on top of Apache APISIX 2.10.0, i.e. 2.10.1, 2.10.2, etc. These releases will backport bugfixes from the main branch.
+
+In October, we plan to release the first patch for the first LTS version, Apache APISIX 2.10.1.
+
+We will then release alternating version lines 2.10.x (e.g. 2.10.2) and 2.x (e.g. 2.11.0) to keep the features iterative while ensuring that the LTS version gets the newer bugfixes.
+
+## New Feature: Add Hosts Attribute to Service
+
+In Apache APISIX 2.10.0, we added the `hosts` property to `service`. Like the other fields in `service`, `route` can inherit the `hosts` attribute from `service`.
+
+For example, the following configurations are equivalent in Apache APISIX 2.10.0:
+
+```json
+# services/1
+{
+    "hosts": ["bar.com"]
+}
+# routes/1
+{
+    "upstream": {
+        "nodes": {
+            "127.0.0.1:1980": 1
+        },
+        "type": "roundrobin"
+    },
+    "service_id": "1",
+    "uri": "/hello"
+}
+```
+
+```json
+# routes/1
+{
+    "upstream": {
+        "nodes": {
+            "127.0.0.1:1980": 1
+        },
+        "type": "roundrobin"
+    },
+    "hosts": ["bar.com"],
+    "uri": "/hello"
+}
+```
+
+With this change, the relationship between `route` and `service` in Apache APISIX is becoming more and more similar to the relationship between `location` and `server` in Nginx. This change brings `service` back into the core Apache APISIX configuration: `route`, `upstream`, and `service`.
```

Comment:
I think we should add `route`, `upstream`, and `service` config docs link.

Reply:
added